src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  color: #fff;
+}
+
+.image {
+  display: block;
+  line-height: 0;
+}
+
+.nav .list {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Change `.nav .list` to use a class instead of the tag name for styling.